### ISSUE TITLE
Delete parsed references from list

### ISSUE THREAD

**Issue by inukshuk** [inukshuk/anystyle#24]
User request:

It would be very handy to have an option to delete an entry entirely from the edit section. Sometimes references are simply broken to begin with but you notice it only after the parsing. For example I just had this bibliographical entry in one of the lists I fed into AnyStyle:

Graddol, David, Dick Leith, et al., eds. Changing English. Milton Park, New York: Routledge  > The spread of English within the British Isles, p. 125 ff.

This entry is irreparably broken, but I cannot take it out of the list without having to parse it again, thus losing all the edits I already made. Now I had to remember, find and delete that entry after the import into Zotero.

**Comment by Frontoish** [comment]
Yes, this would be useful - also in cases where you know it's a duplicate of something you already have and don't want to spend time training the model to deal with it.
